src/mumble/MainWindow.h

```diff
@@ -71,6 +72,7 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		QMenu *qmTray;
 		QIcon qiIcon, qiIconMuteSelf, qiIconMuteServer, qiIconDeafSelf, qiIconDeafServer, qiIconMuteSuppressed;
 		QIcon qiTalkingOn, qiTalkingWhisper, qiTalkingShout, qiTalkingOff;
+		QMap<unsigned int, UserLocalVolumeDialog*> qmuservolTracker;
```

Comment:
s/`qmuservolTracker`/`qmUserVolTracker`/

Use a space between `UserLocalVolumeDialog` and `*`.